package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "test:cypress:open": "cypress open",
+    "test:cypress:open": "cypress open -e grepTags='--@ignore'",
```

Comment:
ignore what?

Reply:
Just means when you use cypress in open mode, it will ignore tests tagged with `@ignore`

Reply:
:+1: 
Based on the flag name it could also appear that it will run the "@ignore" tests exclusively :joy:

Reply:
Can't help the syntax I guess